datacatalog/cloud-client/src/main/java/com/example/datacatalog/CreateFilesetEntry.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.datacatalog;
+
+// [START datacatalog_create_fileset_tag]
```

Comment:
```suggestion
// [START datacatalog_create_fileset]
```

Reply:
The tech writer asked to put _tag at end, because he's using it already that way in the docs... is it ok?

Reply:
The tech writer asked to put _tag at end, because he's using it already that way in the docs... is it ok?

Reply:
If that's what they want that's okay. It just seems a little odd - typically we don't but the word tag as apart of the tag.